modules/run-vault/run-vault

```diff
@@ -37,6 +37,12 @@ function print_usage {
   echo -e "  --s3-bucket\tSpecifies the S3 bucket to use to store Vault data. Only used if '--enable-s3-backend' is set."
   echo -e "  --s3-bucket-region\tSpecifies the AWS region where `--s3-bucket` lives. Only used if `--enable-s3-backend` is set."
   echo
+  echo "Optional Arguments for enabling the AWS KMS seal (Vault Enterprise only):"
+  echo
+  echo -e "  --enable-auto-unseal\tIf this flag is set, enable the AWS KMS Auto-unseal feature. Default is false."
+  echo -e "  --auto-unseal-kms-key\tThe id of the AWS KMS key to be used for encryption and decryption."
```

Comment:
Per above, avoid the word "ID" and instead prefer "name" or "ARN".

Reply:
it is named `ID` all the way. In Amazon AWS key UI, in terraform, in what vault is expecting. Everywhere.

Reply:
Agreed Etiene. Go with ID.